shared/config/curriculum.ts

```diff
@@ -53,6 +53,21 @@ export const languageSuperBlocks = [
   SuperBlocks.A2Chinese
 ];
 
+export enum ChallengeLang {
+  English = 'en-US',
+  Spanish = 'es',
+  Chinese = 'zh-CN'
+}
+
+export const superBlockToLang: Partial<Record<SuperBlocks, ChallengeLang>> = {
```

Comment:
We could consolidate this with the `superBlockToSpeechLang` constant (right below this one).

Reply:
Yeah, I saw it, but I didn't want to touch it on my own.